src/sql/azdata.d.ts

```diff
@@ -26,13 +26,21 @@ declare module 'azdata' {
 			groupId: string;
 			saveProfile: boolean;
 			azureTenantId?: string;
+			options: { [name: string]: any };
```

Comment:
Should we add some documentation for this object? E.g. how it's intended to work, the fact fields are helper accessors over the options, etc.?